Source/Blazorise/Base/BaseComponent.cs

```diff
@@ -491,6 +532,9 @@ public IFluentBorder Border
         get => border;
         set
         {
+            if ( border == value )
+                return;
+
             border = value;
 
             DirtyClasses();

```

Comment:
WE've talked briefly upon this on another PR recently. But all this could be boiled down somehow to a single centralized check in the `SetParametersAsync` where if any of these is changed then it `DirtyClasses`. Probably your new ComponentParameterInfo thingy can help. Or some API that could quickly take in various params and see if anything changed.

Reply:
Yes, not much to say :)